### PR TITLE
refactor: rewrites color definitions to use @atb-as/theme

### PR DESCRIPTION
**Pull request by mikaelbr**
Uses shared styles from `@atb-as/theme` as defined in https://github.com/AtB-AS/design-system.

See the documentation of use for colors and typography: https://github.com/AtB-AS/design-system/tree/main/packages/theme

This PR rewrites ThemeText and Colors to match the new distributed theme module which can be shared by other products.

## TODO

- [x] Rewrite colors
- [x] Rewrite typography names to match new design system names

### CODE REVIEW

**Review by gorandalum — APPROVED** [review]
LGTM! Good work!

File: src/screens/Profile/DesignSystem.tsx [hunk 1]
```diff
@@ -133,12 +133,12 @@ export default function DesignSystem() {
 
         <Sections.Section withPadding withTopPadding>
           <Sections.LocationInput
-            label="Label"
+            label="body__tertiary"
```
Comment (cbrevik) [inline src/screens/Profile/DesignSystem.tsx:136]: Wrongly replaced text here, should be "Label"? Same with next two?

File: src/components/message-box/index.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ const MessageBox: React.FC<MessageBoxProps> = ({
       {retryFunction && (
         <ThemeText
           style={styles.retryText}
-          type="body__link"
+          type="body__primary--underline"
```
Comment (cbrevik) [inline src/components/message-box/index.tsx:62]: Shouldn't try again text have same color as main body text? Contrast issues?
image

**Review by cbrevik — COMMENTED** [review]
Is background_2 supposed to change to current shade? Seems like there's not enough contrast with white section-boxes here. Or is it fine @pettersuul?
image

File: src/screens/Profile/DesignSystem.tsx (hunk 1 above)
Comment (mikaelbr) [reply src/screens/Profile/DesignSystem.tsx:136]: Hah. Fantastic catch! I should have done a case-sensitive replacement AND actually verified. Lazy

**Comment by pettersuul** [comment]
@cbrevik Unsure what values are used here, but background_2 should be darker than background_1 (more contrast)

File: src/components/message-box/index.tsx (hunk 2 above)
Comment (mikaelbr) [reply src/components/message-box/index.tsx:62]: Saw this also. Think it's an issue from a previous PR (@gorandalum ) but can fix it as part of this aswell.

**Comment by mikaelbr** [comment]
Yep. There were duplicated values and I chose the wrong ones. Fixing up now.

File: src/screens/Profile/DesignSystem.tsx (hunk 1 above)
Comment (mikaelbr) [reply src/screens/Profile/DesignSystem.tsx]: Fixed

File: src/components/message-box/index.tsx (hunk 2 above)
Comment (mikaelbr) [reply src/components/message-box/index.tsx:62]: Should be solved now

**Comment by mikaelbr** [comment]
@cbrevik Fixed comments now. Would be great if you could just verify quickly if you still have it locally.

**Review by cbrevik — APPROVED** [review]
Verified fixes, seems good to go! Awesome job :D

**Pull request merged**